src/transformStateWithClones.js

```diff
@@ -7,7 +7,48 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let newState = { ...state };
+  const newObject = [];
+
```

Comment:
Bad naming. newObject but it's an array. 
```suggestion
  const result = [];
```